Windows/Gopher/Gopher/Gopher.cpp

```diff
@@ -171,33 +182,35 @@ void Gopher::handleDisableButton()
 		_disabled = !_disabled;
 
 		if (_disabled) {
-			_controller->Vibrate(10000, 10000);
-			Sleep(400);
-			_controller->Vibrate(0, 0);
+			pulseVibrate(400, 10000, 10000);
 		}
 		else {
-			_controller->Vibrate(65000, 65000);
-			Sleep(400);
-			_controller->Vibrate(0, 0);
+			pulseVibrate(400, 65000, 65000);
 		}
 	}
 }
 
 void Gopher::toggleWindowVisibility()
 {
 	_hidden = !_hidden;
+	setWindowVisibility(_hidden, true);
+}
 
+void Gopher::setWindowVisibility(const bool &hidden, const bool& verbose) const
```

Comment:
I don't like this `verbose` flag. I think it is more *clean* to remove the verbose flag, and leave the function so that it does only *one* thing. Now it is doing two things: setting window visibility **and** printing to console. 

To keep the functionality, you could add the printing code in the `toggleWindowVisiblity` function.